uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -34,7 +34,7 @@ <h4 class="modal-title">
       </ui-select>
     </div>
     <div
-      ng-class="{'chr_disabled': !$ctrl.request.contact_id}"
+      ng-class="{'chr_disabled': (!$ctrl.request.contact_id || $ctrl.loading.postContactSelection)}"
```

Comment:
Instead of using an ad-hoc property (`postContactSelection`) can't you use the `loading.{x}` properties you already have?